files/zh-cn/web/http/status/102/index.md

```diff
@@ -0,0 +1,34 @@
+---
+title: 102 Processing
+slug: Web/HTTP/Status/102
+l10n:
+  sourceCommit: 592f6ec42e54981b6573b58ec0343c9aa8cbbda8
+---
+
+{{HTTPSidebar}}{{deprecated_header}}
+
+
```

Comment:
```suggestion
```

Reply:
Don't need these templates?

Reply:
Only one blank line is needed between paragraphs